ts/pages/staking/wizard/newsletter.tsx

```diff
@@ -82,19 +85,26 @@ const StyledButton = styled(Button)`
 `;
 
 export const Newsletter = () => {
-    const onSubmit = (event: React.FormEvent<HTMLFormElement>): void => {
-        event.preventDefault();
-        alert('submit!');
-    };
+    const [email, setEmail] = React.useState<string>('');
+    const onSubmit = React.useCallback(
+        async (event: React.FormEvent<HTMLFormElement>) => {
+            event.preventDefault();
+            if (!email) {
+                return;
+            }
+            try {
+                await backendClient.subscribeToNewsletterAsync(email);
```

Comment:
NIT: might be worth trimming the email before sending it off to the server, I don't think we do any validation in the backend client
```suggestion
                await backendClient.subscribeToNewsletterAsync(email.trim());
```